docs/spec.md

```diff
@@ -4383,25 +4394,29 @@ In conversations with many stakeholders, this op has come up as effectively
 deprecated, so in the future we are planning to explore removing it
 ([#597](https://github.com/openxla/stablehlo/issues/597)).
 
+For quantized types, performs
+`quantize(rng(dequantize(a), dequantize(b), shape, rng_distribution), type(result))`.
```

Comment:
As discussed in the comment, I think it will be better to scope out `rng`, as the rounding for float-to-quant conversion may change the random variable distribution, which will make this unusable.

Reply:
Thanks this makes sense!
Will be fixed in the next commit.

Reply:
done.